### ISSUE TITLE
SendGrid Accounts API

### ISSUE THREAD

**Issue by jaygridley** [Azure/azure-rest-api-specs#9238]
There is an existing API managing SendGrid accounts, see https://schema.management.azure.com/schemas/2015-01-01/Sendgrid.Email.json#/resourceDefinitions/accounts. We are missing this one in this repository (https://github.com/Azure/azure-rest-api-specs/tree/master/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable). It is affecting other parts of Azure's toolchain as mentioned in https://github.com/Azure/azure-sdk-for-go/issues/8376.

**Comment by atrauzzi** [comment]
Hello :wave:, would love to see this done so that the terraform folks can offer a SendGrid resource!

**Comment by Sharathmk99** [comment]
+1

**Comment by solankisamir** [comment]
ApiManagement does not manage SendGrid Accounts. Removing Label

**Comment by rabidscorpio** [comment]
@solankisamir How about routing this request where it needs to go instead of throwing it in the trash. Which team does manage the sendgrid api?

**Comment by timja** [comment]
@ArcturusZhang would you be able to help with routing this to the right team please?

**Comment by akning-ms** [comment]
+ @devoto13 may know as he is working on schema for sendGrid.

**Comment by devoto13** [comment]
@akning-ms No, not really. I've contributed the schema back in the day as I wanted to create a SendGrid resource using Azure Resource Manager. I don't use it anymore and don't think I can help with routing.

**Comment by frecke** [comment]
+1, needs routing or another look Please
